workshops/templates/workshops/event.html

```diff
@@ -6,6 +6,17 @@
 {% load tags %}
 
 {% block content %}
+
+{% if event.tags_changed %}
```

Comment:
Can `event.metatags_changed` work? We'd call *tags appearing in index.html of each workshop* metatags.